### ISSUE TITLE
Default playback speed shouldn't be a setting

### ISSUE THREAD

**Issue by lu4p** [futo-org/grayjay-android#1356]
### Use Cases

Which playback speed a user is comfortable with may depend on factors like focus, fatigue etc. and may change throughout the day. It should therefore be very easy to change the default playback speed.

A user doing something else while watching videos in the background, may want a lower playback speed.

A user who is very actively paying attention, may want a higher playback speed.

### Proposal

Drop the "default playback speed" setting. Instead the last playback speed set in the video player should be remembered as the "default".

**Comment by MKBontwikkeling** [comment]
Nooo sorry I really like the default playback speed. I can't watch Russel Brand at 2x but many others I can.

**Comment by fcures** [comment]
Or just keep the setting and add another to Use Last Playback Speed, why remobe the whole setting? Downgrading for what purpose?

**Comment by SdoggaMan** [comment]
I agree with the above - it's fine to want to choose a custom, default speed. You may want to stick to the last used speed--I fully agree, this should be an option!--but some people listen to 99% of their YouTube content at 1.25, so a default setting here makes sense. I don't agree that the option should be removed - I think you will _piss off_ far more people by _removing_ it than you will make happy those who don't use it.

**Comment by eklenin** [comment]
Maybe it would be nice to have a UI to "temporarily" change the default playback speed for those who just had two cups of coffee and could use 2x speed for the next 1–2 hours.
